fission-core/library/Fission/Web/Async.hs

```diff
@@ -53,7 +53,7 @@ asyncIdleTimeout ::
   , FromPrefixed prefix Double
   )
   => Seconds prefix n
-  -> Async (Either err a)
+  -> Async a
   -> TVar  (Maybe a)
   -> m (Async (Either Process.TimedOut ()))
 asyncIdleTimeout idleTimeout asyncRef latestVar =
```

Comment:
`Async a` will get inferred to `Async (Either ClientError value)` at the call site. Thanks to the type changing on the class, this function doesn't need to know as much about the innards